src/main/java/org/opensearch/security/auth/AuthFailureListener.java

```diff
@@ -18,12 +18,19 @@
 package org.opensearch.security.auth;
 
 import java.net.InetAddress;
+import java.util.List;
+
+import org.apache.commons.net.util.SubnetUtils.SubnetInfo;
 
 import org.opensearch.security.support.WildcardMatcher;
 import org.opensearch.security.user.AuthCredentials;
 
 public interface AuthFailureListener {
+    List<String> getIgnoreHosts();
```

Comment:
There may be some overlap between this and the WildcardMatcher down below. Do you think we should remove the one below on favor of a single `List<String> getIgnoreHosts();` which covers both static IPs and CIDR blocks?

Reply:
We need both - since WildcardMatcher supports IPs, hostnames and wildcards in IPs & hostnames as well. I can move the IP matching to one method and keep hostname match in another.